glidertest/plots.py

```diff
@@ -38,6 +69,8 @@ def plot_updown_bias(df: pd.DataFrame, ax: plt.Axes = None, xlabel='Temperature
     with plt.style.context(glidertest_style_file):
         if ax is None:
             fig, ax = plt.subplots()
+            third_width = fig.get_size_inches()[0] / 3.11
```

Comment:
I think this is good! Do we want to have the option to have the third/half/full in the parameters instead and put third as default here.
Something like:
```
def plot_updown_bias(df: pd.DataFrame, ax: plt.Axes = None, xlabel='Temperature [C]', size='third', '**kw: dict, ) -> tuple({plt.Figure, plt.Axes}):
if size=='third':
     third_width = fig.get_size_inches()[0] / 3.11
     fig.set_size_inches(third_width, third_width *1.1)
...
```
Or if we want 3 styles and the font size is also adjusted